on-boarding.md

```diff
@@ -29,7 +29,7 @@ One team member will be in charge of supervising the on-boarding process, making
 
 ## On-boarding Supervisor
 
-You are responsible for ensuring our newest team member has a smooth on-boarding experience.
+The on-boarding supervisor is responsible for ensuring our newest team member has a smooth on-boarding experience.
```

Comment:
"Supervisor" feels a little off brand for us... It seems like we would want something a little less "managerial" sounding here and more like, "...everyone plays a part in helping a new team member get started... we always designate at least one teammate as a point of contact... yadda yadda... buddy?... friend?... helper?... facilitator?..." 🤷